admin/worker/delete_expired_token.go

```diff
@@ -0,0 +1,24 @@
+package worker
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+)
+
+func (w *Worker) deleteExpiredTokens(ctx context.Context) error {
+	expiredTokens, err := w.admin.DB.FindExpiredAuthTokens(ctx)
+	if err != nil {
+		return err
+	}
+
+	for _, token := range expiredTokens {
+		w.logger.Info(`deleting expired tokens`, zap.String("id", token.ID), zap.String("expired_ts", token.ExpirationTS.String()))
```

Comment:
The log message should be `deleting expired token` (not plural), `expired_ts` should be `expires_on` and use `zap.Time`, and also add `observability.ZapCtx` as a field